docs/examples/003-area-calc.ipynb

```diff
@@ -0,0 +1,360 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {
+    "collapsed": false
+   },
+   "source": [
+    "# Face Area Calculations\n",
+    "\n",
+    "## Overview\n",
+    "\n",
+    "This notebook will showcase the different area calculation options provided by UXarray\n",
+    "the grid topology attributes and variables stored in the `UXarray.Grid`\n",
+    "object.\n",
+    "\n",
+    "For more details on how to load in data, check out our [previous usage\n",
+    "example](https://uxarray.readthedocs.io/en/latest/examples/read-grid-data.html)\n",
+    "\n",
+    "**This notebook has the following sections:**\n",
+    "1. Compute total area of all faces\n",
+    "2. Options for accuracy of area calculation functions\n",
+    "3. Getting area of all the faces (not total)\n",
```

Comment:
Maybe say "3. Getting individual areas of each face \n" or "3. Getting areas of all the indivdual faces \n" instead?